internal/wts.go

```diff
@@ -65,8 +68,13 @@ type fenceAccessTokenResponse struct {
 
 var myClient = &http.Client{Timeout: 20 * time.Second}
 
-func getJson(url string, target interface{}) (err error) {
-	r, err := myClient.Get(url)
+func getJson(url string, target interface{}, access_token string) (err error) {
+	req, err := http.NewRequest("GET", url, nil)
```

Comment:
Don't have to do this if the rest of the app doesn't, but typically in Go we'd want to use `http.MethodGet`

If we're not doing anything with `err` we may want to replace it with `_`, or at least log it

Reply:
true, good points. i replaced the err with an underscore. it looks like across gen3 fuse, we use `http.NewRequest`, so I'll leave it with that convention for now. making a note to consult the Go stylebook next time we touch FUSE